book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
```

Comment:
- [ ] great job initializing rentals. Please kindly ensure A method for adding rentals should be present.kindly ensur the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. you can refresh on this [lesson](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`)
eg

```ruby
def add_rental(person, date) 
    Rental.new(date, self, person)
  end
```